src/Microsoft.Health.Dicom.Core/Features/Workitem/IAddWorkitemResponseBuilder.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using FellowOakDicom;
+using Microsoft.Health.Dicom.Core.Features.Store;
+using Microsoft.Health.Dicom.Core.Messages.WorkitemMessages;
+
+namespace Microsoft.Health.Dicom.Core.Features.Workitem
+{
+    /// <summary>
+    /// Provides functionality to build the response for the workitem store transaction.
```

Comment:
Update comment to `add workitem transaction`

Reply:
Done.